hydromt_sfincs/sfincs.py

```diff
@@ -1625,13 +1628,7 @@ def setup_waterlevel_forcing(
             )
 
         # set/ update forcing
-        if df_ts is not None:
-            # set timeseries and locations
-            self.set_forcing_1d(df_ts, gdf_locs, name="bzs", merge=merge)
-        else:
-            # set locations only
-            self.set_forcing_1d(gdf_locs=gdf_locs, name="bzs", merge=merge)
```

Comment:
The if-else isn't necessary as set_forcing also works if either df_ts or gdf_locs is None